packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -10,21 +10,60 @@
 @import '~carbon-components/scss/globals/scss/layout';
 @import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
 @import '~carbon-components/scss/globals/scss/css--reset';
+@import '~carbon-components/scss/components/ui-shell/ui-shell';
+@import '~carbon-components/scss/components/ui-shell/side-nav';
 
-/// Masthead styles
 /// @access private
-/// @group masthead
+/// @group dotcom ui-shell
+
+$search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
+$search-transition-timing: 95ms;
+
 @mixin masthead {
   .#{$prefix}--header {
+    transition-timing-function: $search-transition;
     background: $ui-02;
-    border-bottom: 1px solid $ui-01;
+    border-bottom: 1px solid $carbon--gray-20;
+
+    .#{$prefix}--header__logo {
+      padding: 0 $spacing-07;
+      margin-right: $spacing-05;
```

Comment:
Should we be using the `$carbon--grid-gutter` from `@carbon/grid` for these?

Reply:
Hm, I don't know, you think we should? They're not using any grid layouts in the header that I can see. It's just simple spacing.

Reply:
It's my understanding – at least in this scenario – it's spaced out like that in order to maintain grid alignment. If the grid gutter changes for whatever reason using something like `$spacing-05` could leave us hanging. Just something to consider.